Sources/StreamChat/Models/ChatMessage.swift

```diff
@@ -343,6 +343,13 @@ public extension ChatMessage {
             return readByCount > 0 ? .read : .sent
         }
     }
+
+    var isAvailableServerSide: Bool {
+        // If we don't have local state it means the message does not have any pending actions and it is
+        // replicated in both server and local.
+        guard let localState = localState else { return true }
+        return !localState.isLocalOnly
+    }
```

Comment:
Can we make the logic the other way around to make it similar to the` localState.isLocalOnly`? 

```suggestion
    var isLocalOnly: Bool {
        // If we don't have local state it means the message does not have any pending actions and it is
        // replicated in both server and local.
        guard let localState = localState else { return false }
        return localState.isLocalOnly
    }
```

Reply:
Happy to do that